khmer/khmer_args.py

```diff
@@ -19,6 +22,18 @@
 DEFAULT_N_THREADS = 1
 
 
+class _VersionStdErrAction(_VersionAction):
```

Comment:
Sigh. Python 3.3 changed the default action to print version to stdout instead of stderr, since we keep the old behavior this class is needed.